Sources/SaaSquatchWebView/SaaSquatchWebView.swift

```diff
@@ -114,6 +115,20 @@ public class SaaSquatchWebView: WKWebView {
             }
         }
     }
+    
+    public func webView(_ webView: WKWebView,
+                        createWebViewWith configuration: WKWebViewConfiguration,
+                        for navigationAction: WKNavigationAction,
+                        windowFeatures: WKWindowFeatures) -> WKWebView? {
+        if let url = navigationAction.request.url, navigationAction.targetFrame == nil {
+            if url.description.lowercased().contains("http://")
```

Comment:
Shouldn't this be starts with?

Reply:
Even better, now checking the scheme of the parsed url